OgreMain/src/OgreRoot.cpp

```diff
@@ -1465,9 +1467,11 @@ namespace Ogre {
     //-----------------------------------------------------------------------
     void Root::loadPlugin(const String& pluginName)
     {
+      std::cout << "Load: " << pluginName << std::endl;
```

Comment:
Do you want to keep this code change or is it a leftover from a debugging session?

Reply:
Removed.